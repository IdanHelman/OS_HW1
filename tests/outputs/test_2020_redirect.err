cat: new: No such file or directory
cat: new: No such file or directory
cat: new: No such file or directory
smash error: kill: invalid arguments
smash error: kill: invalid arguments
smash error: open failed: No such file or directory
ls: cannot access '>/tmp/smash/test/new': No such file or directory
cat: new: No such file or directory
chmod: cannot access 'new': No such file or directory
smash error: execvp failed: No such file or directory
cat: new: No such file or directory
rm: cannot remove 'new': No such file or directory
smash error: execvp failed: No such file or directory
cat: new: No such file or directory
rm: cannot remove 'new': No such file or directory
ls: cannot access '>new': No such file or directory
cat: new: No such file or directory
