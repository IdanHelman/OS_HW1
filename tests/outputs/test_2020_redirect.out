smash> smash> smash> smash> test_redirect> new
smash> smash> again_redirect> new
smash> smash> test_redirect>> new
smash> smash> smash> smash pid is 5412
smash> smash> smash pid is 5412
smash pid is 5412
smash> smash> smash pid is 5412
smash> mypid
smash> smash> smash> smash> smash> smash> smash> smash> k
mypid
s
t
smash> smash> k
mypid
p
s
t
smash> k
mypid
p
s
t
smash> smash> smash> before
smash> smash> smash> smash> smash> k
mypid
p
s
t
smash> smash> smash> k
mypid
p
s
t
smash> smash> smash> smash> k
mypid
p
s
t
smash> smash> smash> smash> smash> smash> k
mypid
p
s
t
test
smash> smash> 0
2
end: 4
smash> smash> smash> smash: sending SIGKILL signal to 1 jobs:
5464: sleep 100&
